CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## [0.6.0] - 2023-02-21
+
+### Added
+
+- File names and agent address are now validated in config (#39)
+
+### Changed
+
+- Move to go-spiffe/v2 (#25)
```

Comment:
it is in a gray zone, how it is affecting users? is this adding some feature to them?

Reply:
I'm not sure we must include a library upgrade on a changelog unless it affects users in some way

Reply:
i guess, but its the biggest change.

Reply:
agree, but if not affecting to the people using this product, it can cause some confusion when they read the changelog, 
but we can keep this in change log if you prefer, but not adding too much value to users